mobile/lib/screens/explore_screen.dart

```diff
@@ -1156,8 +1156,7 @@ class _ExploreScreenState extends ConsumerState<ExploreScreen>
           await ref.read(popularNowFeedProvider.notifier).refresh();
         } else {
           // For Trending tab, refresh video events
-          ref.invalidate(videoEventsProvider);
-          await ref.read(videoEventsProvider.future);
+          final _ = await ref.refresh(videoEventsProvider);
```

Comment:
Note: Had to assign it to `_` to not get a lint warning that the return value had to be used. But in fact we don't really need to use the returned value since the refresh method will trigger the `watch` methods once it is done.